### ISSUE TITLE
Upgrade issue from 12.0.4 to 13.0.1 with docker image

### ISSUE THREAD

**Issue by meons** [nextcloud/server#8902]
I have tried to upgrade from nextcloud 12.0.4 to 13.0.1 by using the new image and keeping same mounts and data but it fails because of an invalid datetime format as you can see on the attached screen shot.

Need I to perform some changes manually before upgrading ?

### Steps to reproduce
https://github.com/nextcloud/docker#update-to-a-newer-version

### Expected behaviour
Upgrade should be performed

### Actual behaviour
![error](https://user-images.githubusercontent.com/5362939/37665293-3446047c-2c5d-11e8-8a91-e7ed2ce030e3.png)

### Server configuration

**Operating system**:
SLES12SP2, Docker version 17.04.0-ce, build 78d1802

**Web server:**
nextcloud:12.0.4-apache -> nextcloud:13.0.1-apache

**Database:**
10.2.7-MariaDB

**PHP version:**
nextcloud:12.0.4-apache -> nextcloud:13.0.1-apache
**Nextcloud version:** (see Nextcloud admin page)
nextcloud:12.0.4-apache -> nextcloud:13.0.1-apache
**Updated from an older Nextcloud/ownCloud or fresh install:**
nextcloud:12.0.4-apache -> nextcloud:13.0.1-apache

**Comment by BioSLuDge** [comment]
I'm having the same problem except I went from a working 13.0.0 to 13.0.1 image and now I get the same error.  I used docker-compose pull && docker-compose up -d.

**Comment by BioSLuDge** [comment]
Ok so I solved my problem.  I basically ran the following commands:

```
docker-compose down
docker rmi $(docker images --filter "dangling=true" -q --no-trunc)
docker rmi nextcloud_app:latest
docker-compose up -d
```

That caused docker to rebuild my app with the correct version of nextcloud:apache.  When I brought it back up it told me the following:

```
WARNING: Image for service app was built because it did not already exist. To rebuild this image you must use `docker-compose build` or `docker-compose up --build`.  
```

So maybe try that before you cleanse your docker images?  Maybe that's not a bad idea anyway.

But in my docker-compose it says build: app, so isn't that the same thing?  Shouldn't docker-compose build that for us.. it worked fine when I updated to 13.0.0.

I ran pip install --upgrade docker-compose to make sure I had the latest.
 
Anyway, I hope this helps someone else.
